py/visdom/server.py

```diff
@@ -1743,6 +1743,12 @@ def download_scripts(proxies=None, install_dir=None):
                 logging.error('Error {} while downloading {}'.format(
                     exc.reason, key))
 
+    filename = '%s/static/%s/%s' % (install_dir, "js", "MathJax.js")
+    if not os.path.exists(filename):
+        mathjax_js = requests.get("https://raw.githubusercontent.com/mathjax/MathJax/master/es5/tex-mml-svg.js")#("https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_SVG")
```

Comment:
Is it not sufficient to just replace the url in line 1656 with this one?

Reply:
I guess it works.

Reply:
Just give me a little time I will fix this.